papermill/tests/test_parameterize.py

```diff
@@ -1,11 +1,15 @@
-import sys
 import unittest
 
+import six
+
 from ..api import read_notebook
 from ..execute import _parameterize_notebook
 from . import get_notebook_path
 
-PYTHON = 'python2' if sys.version_info[0] == 2 else 'python3'
+if six.PY2:
```

Comment:
nbd but I think the single line version reads better `PYTHON = 'python2' if six.PY2 else 'python3'`

Reply:
👍